bugsnag.php

```diff
@@ -18,7 +18,7 @@ class Bugsnag_Wordpress
     private static $NOTIFIER = array(
         'name' => 'Bugsnag Wordpress (Official)',
         'version' => '1.6.1',
-        'url' => 'https://bugsnag.com/notifiers/wordpress',
+        'docs' => 'https://docs.bugsnag.com/platforms/php/wordpress',
```

Comment:
This key has to be `url` as it's specified in the API:

<img width="738" alt="image" src="https://user-images.githubusercontent.com/282732/149508986-cea7a933-329e-4e92-b122-45f47957c803.png">

The idea of this field isn't really to be user facing, so I don't think it should link to the docs. I would suggest using the repo URL as it's the convention we've gone with on other notifiers recently

```suggestion
        'url' => 'https://github.com/bugsnag/bugsnag-wordpress',
```

Instead of including the docs URL here, the test notification could add it as an additional piece of metadata:

https://github.com/bugsnag/bugsnag-wordpress/blob/6cb64bd54eea412a6e2dac859c72ab35fc19a2db/bugsnag.php#L291

Reply:
Ah of course `$NOTIFIER` is used in constructing the notifier too. I'll add it as additional metadata in that test notification. Thoughts on this were if a user were to install directly through WP plugin browser, they may not yet have seen the docs. 👍

Reply:
Yeah, I agree it makes sense to link to the docs there